src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Write.String.cs

```diff
@@ -18,7 +18,7 @@ public static partial class JsonSerializer
         /// </remarks>
         public static string Serialize<TValue>(TValue value, JsonSerializerOptions? options = null)
         {
-            return WriteCoreString(value, typeof(TValue), options);
+            return Serialize<TValue>(value, typeof(TValue), options);
```

Comment:
I prefer the explict code here since the very next method in the same file has the same name+arguments except the first arg is `object?` instead of TValue